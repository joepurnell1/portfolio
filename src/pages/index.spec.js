import React from "react"
import renderer from "react-test-renderer"
import Index from './index';

describe('index test', () => {
  it('should test something', () => {
    const tree = renderer.create(<Index />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});