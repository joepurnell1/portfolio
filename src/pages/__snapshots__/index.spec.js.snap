// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`index test should test something 1`] = `
<div
  className="sc-bdVaJa hhNaSr"
>
  Hello world!
</div>
`;
